### PR TITLE
Support footnotes and abbreviations

### PR DESCRIPTION
**Pull request by seshrs**
**TLDR:** This PR adds CSS styles to support [footnotes](https://kramdown.gettalong.org/quickref.html#footnotes) and [abbreviations](https://kramdown.gettalong.org/quickref.html#abbreviations).

## Context

While browsing the course website for [CS50 at Tufts University](https://www.cs.tufts.edu/comp/50ISDT/), I was astonished to see that the Markdown files defined footnotes inline with where they were used!

It turns out that kramdown's Markdown syntax [supports adding footnotes](https://kramdown.gettalong.org/quickref.html#footnotes) to pages[^github-footnotes]. This PR adds styles to make footnotes at the end appear in smaller text visually separated from the rest of the content.

[^github-footnotes]: For that matter, did you know that GitHub supports footnotes too? 🤯 

## Validation

Visit https://preview.sesh.rs/previews/eecs485staff/primer-spec/148/ and observe the footnotes and abbreviations on the page.

### CODE REVIEW

**Comment by seshrs** [comment]
Hmm, abbreviations don’t display on mobile. I wonder if we can use tooltips…

**Comment by seshrs** [comment]
After 34f8859, abbreviation expansions appear as tooltips on mobile! 🎉 

However, I noticed that the tooltips get clipped on small screens. I've opened https://github.com/primer/css/issues/1909 to explore potential workarounds.

I'd also like to test how abbreviations and footnotes work on screenreaders.
